packages/merkle-distributor/scripts/1_CreateClaimsForWindow.ts

```diff
@@ -0,0 +1,92 @@
+// Read in a json file containing information for a particular window distribution and produce a claims file containing
+// merkle proofs and a merkle root. This input file is produced from the liquidity/developer/KPI outputs output scripts.
+// The output of this script is then fed into the PublishClaimsForWindow.ts script to add claims to IPFS and Ethereum.
+
+// The input file should implement the following formatting:
+// {
+// "chainId": 42,
+// "rewardToken": "0x47B1EE6d02af0AA5082C90Ea1c2c14c70399186c",
+// "windowIndex": 0,
+// "totalRewardsDistributed": "15000000000000000000",
+// "recipients": {
+//   "0x00b591bc2b682a0b30dd72bac9406bfa13e5d3cd": {
+//     "amount": "1000000000000000000",
+//     "metaData": { "reason": ["YD-WETH-21 Liquidity Mining Week 27"] }
+//   }... for all recipients
+// }
+
+// example execution: ts-node ./scripts/1_CreateClaimsForWindow.ts -i ./scripts/example.json
+
+import assert from "assert";
+import path from "path";
+import { program } from "commander";
+import fs from "fs";
+const { getWeb3 } = require("@uma/common");
+const web3 = getWeb3();
+const { toBN } = web3.utils;
+
+import MerkleDistributorHelper = require("../src/MerkleDistributorHelper");
```

Comment:
nit: why can't this be
```ts
import MerkleDistributor from "../src/MerkleDistributorHelper";
```

or 

```ts
import * as MerkleDistributorHelper from "../src/MerkleDistributorHelper";
```

if both of these don't work for some reason, totally fine to leave as is.